generator/sbpg/targets/resources/sbp_messages_mod.rs

```diff
@@ -22,26 +23,49 @@ use self::(((p.identifier|mod_name)))::(((m.identifier|camel_case)));
 ((*- endfor *))
 use self::unknown::Unknown;
 
-use crate::serialize::SbpSerialize;
+mod lib {
```

Comment:
Should this be called a prelude? Or is that term just for common imports from an external crate?

Reply:
Hmm yeah I just took the idea from serde_json where they call it lib - https://docs.serde.rs/src/serde_json/lib.rs.html#366
Although they are doing it for a slightly different reason (still internal though)